widgetry/src/widgets/mod.rs

```diff
@@ -418,6 +418,21 @@ impl Widget {
         Widget::new(Box::new(Container::new(true, new)))
     }
 
+    /// Creates a row with the specified widgets, with a `spacing` sized margin between members
+    pub fn evenly_spaced_row(widgets: Vec<Widget>, spacing: usize) -> Widget {
```

Comment:
In a followup, I'll change `row` to call this one with the default 10. Another case where default params would be a nice language feature